core/server/worker/src/test/java/alluxio/worker/block/TieredBlockStoreTest.java

```diff
@@ -301,6 +301,22 @@ public void freeSpace() throws Exception {
     assertFalse(FileUtils.exists(BlockMeta.commitPath(mTestDir1, BLOCK_ID1)));
   }
 
+  /**
+   * Tests the {@link TieredBlockStore#freeSpace(long, long, BlockStoreLocation)} method.
```

Comment:
Remove underline?

Reply:
This is annotated by github. There is no underline in the code